console/v1/types_console_link.go

```diff
@@ -19,6 +19,19 @@ type ConsoleLinkSpec struct {
 	Link `json:",inline"`
 	// location determines which location in the console the link will be appended to.
 	Location ConsoleLinkLocation `json:"location"`
+	// applicationMenu holds information about section and icon used for the link in the application menu.
```

Comment:
should say something about only being applicable when the Location is `ApplicationMenu` ?  (it's implied, but not stated explicitly)